modules/accredited_representative_portal/app/services/accredited_representative_portal/representative_user_loader.rb

```diff
@@ -80,5 +81,14 @@ def current_user
 
       @current_user = user
     end
+
+    def verified?
+      # TODO: Should we use the actual logic for `verified?`. The fact that ARP
+      # users are always verified is expressed elsewhere as ARP required LOA
+      # level in ARP `ClientConfig`. An example isntantiation of this is
+      # `db/seeds/development.rb` while it is a DB record in other environments.
+      # session.user_account.verified?
+      true
```

Comment:
@bosawt It seems to me there is a calculation that is genuinely appropriate to make for determining LOA and verified and it just happens to be the case that all our users verified are as a consequence of our client config (is this correct?) and furthermore the single source of truth of that fact is expressed exactly in the client config (again if that's correct). Put another way, if the source of truth changed to enable unverified uses, this code would become logically out of sync.